uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/CRM/HRLeaveAndAbsences/API/Wrapper/LeaveRequestDatesTest.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+use CRM_HRLeaveAndAbsences_API_Wrapper_LeaveRequestDates as LeaveRequestDates;
+
+/**
+ * Class CRM_HRLeaveAndAbsences_API_Wrapper_LeaveRequestDates
+ *
+ * @group headless
+ */
+class CRM_HRLeaveAndAbsences_API_Wrapper_LeaveRequestDatesTest extends BaseHeadlessTest {
+
+  /**
+   * @var \CRM_HRLeaveAndAbsences_API_Wrapper_LeaveRequestDates
```

Comment:
You can use the alias

Reply:
I was sure aliases were not supported in docblocks, but I was just checking [here](https://docs.phpdoc.org/references/phpdoc/types.html#valid-class-name) and it seems they are (assuming that an alias is a "local name"):

> Thus this may be either a Fully Qualified Class Name (FQCN) or if present in a namespace a local name.

I'll update